jenkins_jobs/modules/parameters.py

```diff
@@ -1112,6 +1112,36 @@ def git_parameter_param(registry, xml_parent, data):
     ]
     helpers.convert_mapping_to_xml(pdef, data, mapping, fail_required=True)
 
+def param_separator(registry, xml_parent, data):
+    """yaml: parameter-separator
+    A parameter separator.
+
+    :arg str name: UUID of the separator (optional)
```

Comment:
I'd describe it as: name of the separator (optional, the plugin will assign a randomly generated UUID if not specified)